package.json

```diff
@@ -1,91 +1,34 @@
 {
-  "name": "sage",
-  "version": "10.0.0",
-  "author": "Roots <team@roots.io>",
-  "homepage": "https://roots.io/sage/",
   "private": true,
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/roots/sage.git"
-  },
-  "bugs": {
-    "url": "https://github.com/roots/sage/issues"
-  },
-  "licenses": [
-    {
-      "type": "MIT",
-      "url": "http://opensource.org/licenses/MIT"
-    }
-  ],
-  "browserslist": [
-    "last 2 versions",
-    "android 4",
-    "opera 12"
-  ],
   "scripts": {
-    "prebuild": "npm run -s lint",
-    "build": "webpack --mode development --progress --config resources/assets/build/webpack.config.js",
-    "build:production": "webpack --mode production --progress --config resources/assets/build/webpack.config.js",
-    "build:profile": "webpack --progress --profile --json --config resources/assets/build/webpack.config.js",
-    "start": "node resources/assets/build/server.js --watch",
-    "clean": "rimraf dist",
+    "build": "cross-env NODE_ENV=development webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "build:production": "npm run -s clean && cross-env NODE_ENV=production webpack --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "start": "cross-env NODE_ENV=development webpack --watch --progress --hide-modules --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "hot": "cross-env NODE_ENV=development webpack-dev-server --inline --hot --config=node_modules/laravel-mix/setup/webpack.config.js",
+    "clean": "npm run -s clean:dist",
+    "clean:dist": "rimraf dist",
     "lint": "npm run -s lint:scripts && npm run -s lint:styles",
-    "lint:scripts": "eslint resources/assets/scripts resources/assets/build",
-    "lint:styles": "stylelint \"resources/assets/**/*.{vue,css,sass,scss,sss,less}\"",
+    "lint:scripts": "eslint resources/assets/styles",
+    "lint:styles": "stylelint 'resources/assets/**/*.{vue,css,sass,scss,sss,less}'",
```

Comment:
The reason it uses double quotes https://github.com/roots/sage/pull/1951#issuecomment-323593159 https://github.com/eslint/eslint/commit/6f88546572b6ee8bc93a151f1df2b4f85c2abafe

Also a typo `resources/assets/styles` -> `resources/assets/scripts` in the `lint:scripts` task

Reply:
Thanks for pointing that out. Will fix it in my next push.